mis_builder_budget/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     'name': 'MIS Builder Budget',
```

Comment:
no extra dependency? Or `analytic` comes from `mis_builder`? (I'd add it anyway to make it explicit)

Reply:
@simahawk yup, this module depends on `analytic` indirectly:
`this_module` -> `mis_builder` -> `account` -> `analytic`